cpp/src/utilities/spmv_1D.cuh

```diff
@@ -17,50 +17,44 @@
 #pragma once
 #include <rmm/thrust_rmm_allocator.h>
 #include <raft/handle.hpp>
-#include "utilities/cusparse_helper.h"
-// FIXME : use RAFT cusparse wrapper
-// The wrapper is about to be upgraded in RAFT
-// better wait for it to be stable before building on top of it
-// #include <raft/sparse/cusparse_wrappers.h>
 #include "utilities/error.hpp"
 
 namespace cugraph {
 namespace opg {
 
-template <typename VT, typename ET, typename WT>
+template <typename vertex_t, typename edge_t, typename weight_t>
 class OPGcsrmv {
  private:
   size_t v_glob;
   size_t v_loc;
   size_t e_loc;
-  const raft::comms::comms_t& comm;
-  VT* part_off;
-  VT* local_vertices;
+  raft::comms::comms_t const& comm;

```

Comment:
We're storing a reference to the handle in line 43 (https://github.com/rapidsai/cugraph/pull/1019/files#diff-da69c92a20221e39f984232cbe12b095R43). As the handle has a pointer to the communicator, should we store references to both handle and communicator?

Reply:
Fixed.